### ISSUE TITLE
CicerMine: need gfa files for echinospermum and reticulatum

### ISSUE THREAD

**Issue by sammyjava** [legumeinfo/mine-issues#45]
Here's the new LIS-style files I've put under /data/v2, lacking GFAs:
```
Cicer/echinospermum/annotations/
└── S2Drd065.gnm1.ann1.YZ9H
    ├── cicec.S2Drd065.gnm1.ann1.YZ9H.cds.fna
    ├── cicec.S2Drd065.gnm1.ann1.YZ9H.gene_models_main.gff3
    ├── cicec.S2Drd065.gnm1.ann1.YZ9H.protein.faa
    └── cicec.S2Drd065.gnm1.ann1.YZ9H.transcript.fna

Cicer/reticulatum/annotations/
└── Besev079.gnm1.ann1.F01Z
    ├── cicre.Besev079.gnm1.ann1.F01Z.cds.fna
    ├── cicre.Besev079.gnm1.ann1.F01Z.gene_models_main.gff3
    ├── cicre.Besev079.gnm1.ann1.F01Z.protein.faa
    └── cicre.Besev079.gnm1.ann1.F01Z.transcript.fna
```

**Comment by adf-ncgr** [comment]
thanks for the reminder about the GFAs. I've got them running now should have them later tonight (not that I expect you to receive them then). And don't forget to include Cicer/arietinum/annotations -- CDCFrontier.gnm3.ann1.KEY4 in the set to be loaded.

**Comment by sammyjava** [comment]
Yup, doing the new arietinum, echinispermum, reticulatum genomes/annotations. Not including the older arietinum on the LIS DS.

**Comment by adf-ncgr** [comment]
OK great, and I've put the gfa files into place- appropriately named in the locations under /erdos/legumeinfo/data/v2/Cicer/ that you created. Note that the folders and files there are all owned by sysadm so I had to sudo cp them in, I think I had noticed this when you had done some sort of mass copy earlier (before your hiatus) and I just did a massive chown -R then. Not sure if you were intending the permissions to be restrictive as a barrier to unintentional havoc or that just happened for some other reason. Anyway, we should probably figure out how to manage that "mirror-like" structure so we can all work there without stepping on each others' toes.
